SECURITY.md

```diff
@@ -5,3 +5,77 @@ As with any complex system, it is certain that bugs will be found, some of them
 ## Public Disclosure Timing
 
 The public disclosure date is agreed between the Cortex Team and the bug submitter. We prefer to fully disclose the bug as soon as possible, but only after a mitigation or fix is available. We will ask for delay if the bug or the fix is not yet fully understood or the solution is not tested to our standards yet. While there is no fixed timeframe for fix & disclosure, we will try our best to be quick and do not expect to need the usual 90 days most companies ask or. For a vulnerability with a straightforward mitigation, we expect report date to disclosure date to be on the order of 7 days.
+
+---------------------------
+
+## Private Vendors List
+
+Cortex is being used by several vendors to provide a hosted Prometheus experience
+to their users. We also have list that is used to provide actionable information
+to multiple vendors at once. This list is not intended for individuals to find out about
+security issues.
```

Comment:
I find this wording a bit confusing.  How about replacing the whole paragraph with
"We also have a list for vendors who provide Cortex to their users."